app/config/collections.php

```diff
@@ -2424,7 +2424,7 @@
                 '$id' => ID::custom('_key_entrypoint'),
                 'type' => Database::INDEX_KEY,
                 'attributes' => ['entrypoint'],
-                'lengths' => [2048],
+                'lengths' => [768],
```

Comment:
Hmm.. These seem like magic numbers for anyone referring to it in the future. 
Can we have constants in the DB library to represent these key lengths ?

Reply:
That's is true 
I will have to make another tag for this